--- OilWar-1.1.0/Makefile.am~	Mon Mar 10 13:43:40 2003
+++ OilWar-1.1.0/Makefile.am	Mon Mar 10 13:44:11 2003
@@ -7,7 +7,7 @@
 datafile_DATA = background.png bitmaps.png\
 singlegunshot.wav bulletricochet.wav explode.wav gasp.wav
 
-scorefiledir = /var/lib/games
+scorefiledir = /var/games
 scorefile_DATA = oilwar.scores
 
 AM_CPPFLAGS = $(sound_flag) -DDATA_DIR=\"$(datafiledir)/\" -DSCORE_DIR=\"$(scorefiledir)/\"
